packages/lib/src/vis/shared/VisCanvas.module.css

```diff
@@ -68,10 +69,9 @@
 }
 
 .floatingToolbar {
-  position: absolute;
-  right: 0;
-  bottom: 0;
-  display: flex;
+  grid-area: canvas;
+  justify-self: end;
+  align-self: end;
   z-index: var(--h5w-zi-floatingToolbar);
-  pointer-events: auto;
```

Comment:
No longer needed, since `floatinToolbar` is no longer inside `r3fRoot`/`canvasWrapper`, so `pointer-events` are never disabled.